firebase-installations/src/androidTest/java/com/google/firebase/installations/FakeCalendar.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations;
+
+import java.util.Calendar;
+
+public class FakeCalendar extends Calendar {
+  private long timeInMillis;
+
+  public FakeCalendar() {
+    timeInMillis = 5000000;
```

Comment:
epochInMillis?

Maybe consider passing the fake current time in via the constructor?

Reply:
Added the start time as a constructor param.

I think "timeInMillis" is more accurate, or timeSinceEpochInMillis.